v2x/vlog_to_pbtype.py

```diff
@@ -54,6 +54,27 @@
     - `(* carry = ADDER *)` : specify carry chain pack_pattern associated
                               with this port
 
+The following attributes are used to annotate cells with fasm metadata:
+    - `(* FASM_PREFIX="prefix" *)` : Sets the fasm prefix of a cell instance.
+        Cannot be set on a module definition!
+
+    - `(* FASM_PREFIX="prefix1;prefix2;..." *) : Same as FASM_PREFIX but used
+        for cells inside `generate` statements. Each instance of a cell will be
+        assigned one prefix from the semicolon separated list. The number of
+        prefixes in the list must match the cell instance count!
+
+    - `(* FASM_FEATURES="str1;str2;..." *)` : Specifies a list of fasm features
+        for the given module. Has to be provided at a module definition.
+        Multiple features have to be separated by semicolon.
+
+    - `*( FASM_FEATURES_{mode}="str1;str2;..." *)` : Same as above but defines
+        a list of fasm features only for the given mode. Those will be attached
+        to the pb_type corresponding to that mode.
+
+    - `(* FASM_PARAMS="str1=param1;str2=param2;.." *)` : A semicolon separated
```

Comment:
I think we need to rework how this one works.

Reply:
That's something that I'm not happy about either. I'll try the idea with Surelog integration to get attributes on parameters.

Reply:
@mkurc-ant - ping?

Reply:
This is fixed, Surelog is used to get attributes from parameters.